test/src/timer.c

```diff
@@ -28,6 +28,11 @@ timer_ratio(timedelta_t *a, timedelta_t *b, char *buf, size_t buflen) {
 	size_t i = 0;
 	size_t j, n;
 
+	/* Bump t1 if it is 0 to avoid dividing 0. */
+	if (t1 == 0) {
+	    t1 = 1;
```

Comment:
The number is odd enough to raise the attention from the user, is any extra warning needed?

Reply:
Let's certainly add more comments, e.g.

The time difference could be 0 if the two clock readings are identical, either due to the operations being measured in the middle took very little time (or even got optimized away), or the clock readings are bad / very coarse grained clock.

As for the warning output, no strong opinion either way. When it happens, the results would look obviously wrong, is that so?

Reply:
Yes, it should look obviously wrong, an example should look like this:
`100000000 iterations, malloc_free_array=1320029us (13.200 ns/iter), delete_array=0us (0 ns/iter), time consumption ratio=1320029:1`